lib/checks/color/link-in-text-block.json

```diff
@@ -1,14 +1,20 @@
 {
   "id": "link-in-text-block",
   "evaluate": "link-in-text-block-evaluate",
+  "options": {
+    "requiredContrastRatio": 3
+  },
   "metadata": {
     "impact": "serious",
     "messages": {
       "pass": "Links can be distinguished from surrounding text in some way other than by color",
-      "fail": "Links need to be distinguished from surrounding text in some way other than by color",
+      "fail": {
+        "fgContrast": "The link has insufficient color contrast of ${data.contrastRatio}:1 with the surrounding text. (Minimum contrast is ${data.requiredContrastRatio}:1, link text: ${data.nodeColor}, surrounding text: ${data.parentColor})",
```

Comment:
A thought that just occurred to me is that these messages mention color contrast when what we're actually analyzing is relative luminance contrast. Worth capturing in the error message to avoid confusion?